core/bones/text.py

```diff
@@ -283,9 +283,10 @@ def __init__(
     def singleValueSerialize(self, value, skel: 'SkeletonInstance', name: str, parentIndexed: bool):
         return value
 
-    def singleValueFromClient(self, value, skel, name, origData):
-        err = self.isInvalid(value)  # Returns None on success, error-str otherwise
-        if not err:
+    def singleValueFromClient(self, value: Any, skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
```

Comment:
Is it necessary to provide full type annotations in sub-classes as well?
I would like to keep maintenance overhead low, if possible.